feat(Images): Add asset image support for wysiwyg node type

mjoellnier opened the pull request:
Hey there! 

I added a recognition of Image tags in the wysiwyg value. So when an cockpit asset is used the image url gets replaced by the fetched one. External images stay untouched. 

👋

mjoellnier committed:
feat(Images): Add asset image support for wysiwyg node type

Changed img tag recognition and Url vs asset

Test commit

Revert "Test commit"

This reverts commit 084c855ce98c6845488ea1c7a20e1bc23378aa5c.

Implemented review feedback

WhippetsAintDogs reviewed (changes requested):
Hi @mjoellnier ! Thanks for your contribution and sorry for the delays.

Almost everything seems fine, I've added refactor suggestions to apply before integrating into master (for a more consistent code in regard to the rest of the project).

The only problem remaining is that currently, in order for your code to work, the inserted image in the Wysiwyg has to  be included at the same time by some other nodes of type `Asset`, `Image` or `Gallery` because it isn't pulled locally otherwise (and in such cases, the expression `images[sources[index]]` would be `undefined`). Have a look at the `normalizeResources` method of the `CockpitService` class in order to understand how to eventually add and link a `normalizeNodeItemWysiwygs` method.

Also, could you:
- squash your two commits into one (keeping the last one's name);
- rebase onto latest master;
- bump `package.json` version to 1.1.2;
- update `package.json` description with: `"This is a Gatsby version 2.x.x source plugin that feeds the GraphQL tree with Cockpit Headless CMS collections and singletons data. Actually, it supports querying raw texts (and any trivial field types), Markdown, Wysiwyg, images, galleries, assets, sets, repeaters, layout(-grid)s (currently only without nested images/assets), objects, linked collections and internationalization."`;
- update `readme.md` description on line 5 with: `Actually, it supports querying raw texts (and any trivial field types), Markdown, Wysiwyg, images, galleries, assets, sets, repeaters, layout(-grid)s (currently only without nested images/assets), objects, linked collections and internationalization.`;
- update `readme.md` by adding that section below the `#### Markdowns` section:
```
#### Wysiwygs

Wysiwyg fields (or `What you see is what you get`) works the same way as Markdown fields, but instead of accessing an HTML equivalent through the `value → childMarkdownRemark → html` attribute, it's accessible straight through the `value` attribute.
```

mjoellnier commented:
Hi @WhippetsAintDogs 
Thanks a lot for your feedback 🤗 I will have a look at it as soon as possible and will let you know!

mjoellnier committed:
Merge branch 'master' of https://github.com/fikaproductions/fika-gatsby-source-cockpit into fikaproductions-master

mjoellnier committed:
Merge branch 'fikaproductions-master'

mjoellnier commented:
Hi @WhippetsAintDogs 

thank you again for your feedback! I implemented the missing parts and added your refactoring suggestions. So please recheck the code - I tested it locally and it works well with images that are only added via WYSIWYG. 

Take care 👋
